PhotoScraper/RPI_SIS_PhotoScraper.py

```diff
@@ -115,14 +117,23 @@ def saveImagesToFolder(term, course, class_list):
             if k == "img url":
                 img_url = class_list[i].get(k)
         # download and save the image to a specific folder (term/course_section) from the image url
-        img_name = rcs_id+".png"
-        filepath = path / img_name
         if img_url.split("/")[-1].strip() == "web_transparent.gif":
             print("Skipping {} because no photo on SIS".format(rcs_id))
             continue
         r = requests.get(img_url)
+
+        #Open the image use PIL.Image to deduce the extension
+        img = Image.open(BytesIO(r.content))
+        img_name = rcs_id
+        if img.format == "JPEG":
+            img_name = img_name + ".jpg"
+        else:
+            img_name = img_name + "." + img.format.lower()
```

Comment:
Use [imghdr](https://docs.python.org/3/library/imghdr.html) instead of making a dependency for pillow:
```
img_name = rcs_id + '.' + imghdr.what(None, BytesIO(r.content))
```